### PR TITLE
Updated frontend: fixed debug port not read from URI.

### PR DESCRIPTION
**Pull request by marcominetti**

### CODE REVIEW

File: front-end-node/NodeInspectorOverrides.js [hunk 1]
```diff
@@ -58,6 +60,18 @@ WebInspector.NodeInspectorOverrides.prototype = {
       showMainAppFile,
       null
     );
+  },
+  
+  _mergeConnectionQueryParams: function(){
+    var RuntimeQueryParamsObject = Runtime._queryParamsObject;
+    if (RuntimeQueryParamsObject['ws'] && RuntimeQueryParamsObject['port']){
+      if (RuntimeQueryParamsObject['ws'].indexOf('?') === -1){
```
Comment (3y3) [inline front-end-node/NodeInspectorOverrides.js]: Is there a situation then `ws` param contains `?`

**Comment by 3y3** [comment]
Please fix brackets style (missed spaces).
We use next syntax in node-inspector:
```js
if (statement) {

}

function(args) {

}
```

I don't see the reason to use `RuntimeQueryParamsObject ` (one long name instead of one other).
It may be reasonable to use short and readable `params`.

**Comment by marcominetti** [comment]
Ok ;)

File: front-end-node/NodeInspectorOverrides.js (hunk 1 above)
Comment (marcominetti) [inline front-end-node/NodeInspectorOverrides.js]: I don't know but some plugins could extend it, right? I would add it just to be safe... ;)

**Comment by 3y3** [comment]
Looks better. I'm ready to merge this one. Please squash changes.

**Comment by marcominetti** [comment]
squashed

**Comment by 3y3** [comment]
Landed as 88921eb8d097df9e20864fa5b0a38c0e0bd7bbc6

**Pull request closed without merging**